### ISSUE TITLE
Feature Request - edit parameter inputs after submitting plan

### ISSUE THREAD

**Issue by jvrana** [klavinslab/aquarium-bugs#298]
## Feature request

As a researcher, it would be nice to be able to edit parameter inputs after a plan is submitted in the same way that you can edit items after a plan is submitted.

Since parameters don't depend on anything, it would be nice to be able to edit parameter inputs on any operations in a plan from leafs all the way up to the goal operations

**Comment by klavins** [comment]
I agree, that would be a good fix. I'll try to figure out an elegant way to do that. Right now the operation i/o form and the operation info display are mutually exclusive.
